examples/timestamp/signed_timestamp.c

```diff
@@ -36,67 +36,48 @@
 /* --- BEGIN TPM Timestamp Test -- */
 /******************************************************************************/
 
-typedef struct tpmKey {
-    TPM_HANDLE          handle;
-    TPM2B_AUTH          auth;
-    TPM2B_PRIVATE       priv;
-    TPM2B_PUBLIC        pub;
-    TPM2B_NAME          name;
-} TpmKey; /* Type used to store the output from Key generation */
-
-
 int TPM2_Timestamp_Test(void* userCtx)
 {
     int rc;
-    TPM2_CTX tpm2Ctx;
+    WOLFTPM2_DEV dev;
     TPMS_TIME_ATTEST_INFO* attestedTime = NULL;
 
     union {
-        GetTime_In getTime;
-        /* For creating keys */
-        CreatePrimary_In createPri;
-        Create_In create;
         /* For managing TPM session */
         StartAuthSession_In authSes;
         PolicySecret_In policySecret;
-        /* Loading a key and removing objects */
-        Load_In load;
+        /* For removing keys after use */
```

Comment:
It looks like you can remove `cmdIn` and `cmdOut` now.

Reply:
Hm, I thought I did. Strange. Maybe between different branch versions it remained. Thanks for catching this leftover.

Reply:
Wait @dgarske  I have removed all possible cmdIn and cmdOut. I can not remove it completely, because we need it for the policySecret, authSesssion (creating and flushing it).